Test test_rbd_thick_provisioning failing due to timeout creating PVC

The test case 
tests/manage/pv_services/test_rbd_thick_provisioning.py::TestRbdThickProvisioning::test_rbd_thick_provisioning
is failing due to timeout while creating the PVC. 

```
>           raise ResourceWrongStatusException(resource.name, resource.describe())
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource pvc-test-189e532fab004bdea49ffbfa4b157e7 describe output: Name:          pvc-test-189e532fab004bdea49ffbfa4b157e7

E             Normal   Provisioning          0s (x3 over 2m31s)   openshift-storage.rbd.csi.ceph.com_csi-rbdplugin-provisioner-8448db4dc9-kmtck_c7276c97-67c3-40f5-b72d-7411ae591dfe  External provisioner is provisioning volume for claim "namespace-test-5a4ae3aaa9ab4e65bad648dce/pvc-test-189e532fab004bdea49ffbfa4b157e7"

ocs_ci/helpers/helpers.py:120: ResourceWrongStatusException
```
Test report : https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/907/testReport/

The PVC is not reaching Bound state within the given time of 150 seconds. This  error is also seen in another test run.

From csi-provisioner logs of csi-rbdplugin-provisioner  pod it is observed that the provisioning took 213 seconds. 
```
2021-05-20T18:29:42.686813239Z I0520 18:29:42.686750       1 controller.go:1335] provision "namespace-test-5a4ae3aaa9ab4e65bad648dce/pvc-test-189e532fab004bdea49ffbfa4b157e7" class "storageclass-test-rbd-92ed1da44c154677b8": started
2021-05-20T18:33:15.481514862Z I0520 18:33:15.481502       1 controller.go:1459] provision "namespace-test-5a4ae3aaa9ab4e65bad648dce/pvc-test-189e532fab004bdea49ffbfa4b157e7" class "storageclass-test-rbd-92ed1da44c154677b8": succeeded
```

--- a/tests/manage/pv_services/test_rbd_thick_provisioning.py
+++ b/tests/manage/pv_services/test_rbd_thick_provisioning.py
@@ -65,7 +65,7 @@ class TestRbdThickProvisioning(ManageTest):
                 access_modes=access_modes_rbd,
                 status=constants.STATUS_BOUND,
                 num_of_pvc=3,
-                timeout=150,
+                timeout=300,
             )
             for pvc_obj in pvc_objs:
                 pvc_obj.io_file_size = pvc_and_file_sizes[pvc_size]

